internal/cloudapi/v2/v2.go

```diff
@@ -205,6 +205,22 @@ func (h *apiHandlers) PostCompose(ctx echo.Context) error {
 		}
 	}
 
+	if request.Customizations != nil && request.Customizations.Filesystem != nil {
+		var fsCustomizations []blueprint.FilesystemCustomization
+		for _, f := range *request.Customizations.Filesystem {
+
+			fsCustomizations = append(fsCustomizations,
+				blueprint.FilesystemCustomization{
+					Mountpoint: f.Mountpoint,
+					MinSize:    f.Size,
```

Comment:
Same problem as above, `f.Size` is `*int`, but `MinSize` expects `uint`.

Reply:
Ah, actually it expects `uint64`. So to make this work you'll need to cast it: `uint64(f.MinSize)`.

This would be bad in case `MinSize` was passed in as a negative number (it would be turned into a huge positive one). Though, we need to guard against too large filesystem sizes anyway, so this might be acceptable. I would leave it as is, but maybe in a follow-up you could add a minimum size (of 0) to the openapi spec, then if we validate the requests against the spec we'll be fine.

Reply:
makes sense, will do!